package.json

```diff
@@ -18,6 +18,7 @@
     "@mui/icons-material": "^5.15.18",
     "@mui/material": "^5.15.18",
     "@mui/x-date-pickers": "^7.7.0",
+    "@mui/x-date-pickers-pro": "^7.8.0",
```

Comment:
lets remove this one since we wont be using it

Reply:
Fixed